test/integration/scheduled_stop_test.go

```diff
@@ -154,6 +154,10 @@ func ensureMinikubeStatusStopped(ctx context.Context, t *testing.T, profile stri
 		if got != state.Stopped.String() {
 			return fmt.Errorf("expected post-stop host status to be -%q- but got *%q*", state.Stopped, got)
 		}
+		got = Status(ctx, t, Target(), profile, "TimeToStop", profile)
+		if got != "Nonexistent" {
```

Comment:
Instead of making sure that `TimeToStop` is nonexistant after stop, let's make sure it's set to something after we schedule the stop but before it happens.

That could happen here:

https://github.com/kubernetes/minikube/blob/master/test/integration/scheduled_stop_test.go#L54